.cirrus.yml

```diff
@@ -325,7 +327,8 @@ testing_task:
             # Images are generated separately, from build_images_task (below)
             image_name: "${FEDORA_CACHE_IMAGE_NAME}"
             image_name: "${PRIOR_FEDORA_CACHE_IMAGE_NAME}"
-            image_name: "${UBUNTU_CACHE_IMAGE_NAME}"
+            # TODO: image_name: "${UBUNTU_CACHE_IMAGE_NAME}"
```

Comment:
Why not?

Reply:
Sorry, the reason is in the commit message (multiple test failures on Ubuntu 19).
I'll add a comment about this.  It can be fixed and enabled in a future PR.

Reply:
okay, comment added in a6534a7 in case others have this same question.